local-running/setSsmConfigForLocalOrCloudRunning.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
```

Comment:
I don't know if any of these config values are particularly sensitive, but would it be better to put this file in the core-common-infra repo?

Reply:
No, they're not sensitive. And if this way of running core was actually accepted then it may make sense to put them in common-infra. But for a spikey thing that will probably die a death this seems like a reasonable way to optionally add them to your dev params.